src/Alert/index.jsx

```diff
@@ -63,7 +64,9 @@ Alert.propTypes = {
   /** Callback for user dismissal actions */
   onUserDismiss: PropTypes.func,
   /** Variant of Alert to use */
-  kind: PropTypes.oneOf(["info", "error", "success"]),
+  kind: PropTypes.oneOf(["info", "error", "success", "plain"]),
```

Comment:
Based on conversations with design, this new type of alert should be `info`, and the idea was to add `warn` to have the styles of the current `info` Alert.

```suggestion
  kind: PropTypes.oneOf(["info", "error", "success", "warn"]),
```

Reply:
Yeah - I was thinking that naming would make more sense, but didnt want to change the current naming convention to break anything. I'll look around the repo when I bump the NDS version to make sure that we change the kind of any alerts that need it.